chapter01/Intro_example_short.ipynb

```diff
@@ -0,0 +1,671 @@
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In CUQIpy, we can define the likelihood function as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "likelihood = b(b=b_obs)\n",
+    "print(likelihood)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the likelihood function is a density function and is not a distribution. If we try to compute `pdf` for example, we will get an error:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We plot the likelihood function for the observed data `b_obs`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x1_lim = np.array([-5, 5])\n",
+    "x2_lim = np.array([-5, 5])\n",
+    "plot_pdf_2D(\n",
+    "    likelihood,\n",
+    "    x1_min=x1_lim[0], x1_max=x1_lim[1],\n",
+    "    x2_min=x2_lim[0], x2_max=x2_lim[1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note:** One method of estimating the inverse proble solution is to maximize the likelihood function, which is equivalent to minimizing the negative log-likelihood function. This is known as the maximum likelihood (ML) point estimate.\n",
+    "\n",
+    "For this problem, all the points $(x_1, x_2)$ that satisfy $x_1 + x_2 = b^\\mathrm{obs}$ are solutions to this maximization problem. These solutions are depicted as the red line in the likelihood plot below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot the likelihood\n",
+    "plot_pdf_2D(\n",
+    "    likelihood,\n",
+    "    x1_min=x1_lim[0], x1_max=x1_lim[1],\n",
+    "    x2_min=x2_lim[0], x2_max=x2_lim[1])\n",
+    "\n",
+    "# Plot the line x2 = b_obs - x1\n",
+    "plt.plot(x1_lim, b_obs-x1_lim, '--r')\n",
+    "plt.ylim(x2_lim);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Combining the likelihood with the prior will give us a unique maximum a posteriori (MAP) point estimate as we will see next."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <font color=#8B4513> 1.6. Putting it all together, the BIP </font> <a class=\"anchor\" id=\"r-putting-it-all-together-the-bip\"></a>\n",
+    "\n",
+    "#### Posterior definition using the Bayes’ rule\n",
+    "- The posterior is proportional to the product of **likelihood** and **prior**\n",
+    "\n",
+    "$$\\pi(\\mathbf{x} | b) \\propto \\pi( b|\\mathbf{x})\\pi(\\mathbf{x})$$\n",
+    "\n",
+    "- Note that $\\pi( b|\\mathbf{x})$ here denotes the likelihood and not the data distribution, despite often written that way.\n",
```

Comment:
$b$ here and in the equation above is actually $b_{obs}$, right? if we stick to the notation introduced...

Reply:
you are right, I changed the writing a bit to clarify that